SCClassLibrary/Common/Control/Server.sc

```diff
@@ -376,7 +376,13 @@ Server {
 		};
 	}
 
-	/* id allocators */
+	/* clientID */
+
+	userSpecifiedClientID {
+		this.deprecated(thisMethod);
+		"Server:clientID can now be set while a server is off, and is locked while the server is running. Thus, userSpecifiedClientID is no longer needed internally, and meaningless. Returns false just in case user code calls this.".postln;
```

Comment:
You should put this in the actual deprecations file:

https://github.com/supercollider/supercollider/blob/43fa8855735cdc914261311b2fe09834db440ab4/SCClassLibrary/deprecated/3.9/deprecated-3.9.sc

If you want, you could put this info in the documentation, but posting it here as a string is not really typical usage, so I'd recommend against it. I think we have an issue open somewhere about extended `DeprecatedError`.